packages/x-charts-pro/src/internals/plugins/useChartProZoom/gestureHooks/usePanOnDrag.ts

```diff
@@ -47,41 +46,47 @@ export const usePanOnDrag = (
   // Add event for chart panning
   React.useEffect(() => {
     const element = svgRef.current;
+    let isInteracting = false;
+    const accumulatedChange = { x: 0, y: 0 };
 
     if (element === null || !isPanOnDragEnabled) {
       return () => {};
     }
 
     const handlePanStart = (event: PanEvent) => {
       if (!(event.detail.target as SVGElement)?.closest('[data-charts-zoom-slider]')) {
-        startRef.current = store.state.zoom.zoomData;
+        isInteracting = true;
       }
     };
     const handlePanEnd = () => {
-      startRef.current = null;
+      isInteracting = false;
     };
 
-    const throttledCallback = rafThrottle((event: PanEvent, zoomData: readonly ZoomData[]) => {
-      const newZoomData = translateZoom(
-        zoomData,
-        { x: event.detail.activeDeltaX, y: -event.detail.activeDeltaY },
-        {
-          width: drawingArea.width,
-          height: drawingArea.height,
-        },
-        optionsLookup,
+    const throttledCallback = rafThrottle(() => {
+      const x = accumulatedChange.x;
+      const y = accumulatedChange.y;
+      accumulatedChange.x = 0;
+      accumulatedChange.y = 0;
```

Comment:
I think this works, but I'm not sure it's very logical. 

IMO, it would be make more sense for `throttledCallback` to accept the `accumulatedChange` object and for `handlePanStart` to create a new `{ x: 0, y: 0}` object when it's fired.

We don't need to reset values inside the throttled function.